packages/components/src/number-control/index.js

```diff
@@ -169,8 +175,9 @@ export function NumberControl(
 			max={ max }
 			min={ min }
 			ref={ ref }
+			step={ jumpStep }
 			type={ typeProp }
-			value={ initialValue }
+			value={ valueProp }
```

Comment:
I've tracked the use of this by the sub-layers and I haven't seen any change to how it's dealt with. However, before this PR the value was normalized to a floating-point number. Would it be a problem that we don't do it now?

Reply:
I just did a couple of tests.

With how the "pass-through" of props->state works now, normalizing the value is disruptive. This is applicable only when the `type` is `text` for this component. In the case of `UnitControl`, it uses  this component underneath, but it takes care of normalizing values.

This is a little tricky to describe. Hopefully that makes sense!

In other words, it's currently structured where validation/normalization comes from the consumer, rather than the component itself. (Secretly, I'm hoping to enhance this in the future so that there's less work for consumers).

For now, I don't think there is a problem, based on currently use-cases and implementations.

Reply:
:+1:

So, if I'm an external (non-Gutenberg) consumer of this component, now I need to do the normalization myself, right? I guess this also needs a place in the changelog. As I mentioned in the other thread above, I'm unsure about what/how we log things in the changelog for this particular package, but IMHO, this should be logged as a breaking change.